package.json

```diff
@@ -38,14 +38,14 @@
     "ember-disable-prototype-extensions": "^1.0.0",
     "ember-disable-proxy-controllers": "^1.0.0",
     "ember-export-application-global": "^1.0.3",
-    "ember-try": "0.0.6",
-    "ember-weakmap": "0.3.0"
+    "ember-try": "0.0.6"
   },
   "keywords": [
     "ember-addon"
   ],
   "dependencies": {
-    "ember-cli-babel": "^5.1.3"
+    "ember-cli-babel": "^5.1.3",
+    "ember-weakmap": "0.4.0"
```

Comment:
weakmap needs to be a dependency or an error will be thrown when the consumer uses the addon